src/WSSecurity/KeyIdentifier/BinarySecurityToken.php

```diff
@@ -0,0 +1,23 @@
+<?php
+declare(strict_types=1);
+
+namespace Soap\Psr18WsseMiddleware\WSSecurity\KeyIdentifier;
+
+use DOMElement;
+use RobRichards\WsePhp\WSSESoap;
+use Soap\Psr18WsseMiddleware\WSSecurity\KeyStore\KeyInterface;
+use VeeWee\Xml\Dom\Document;
+
+class BinarySecurityToken implements KeyIdentifier
```

Comment:
Should adding the binary token go in a separate entry? In soap-ui it is more like a selection of the key identifier.
Not sure if this is reusable for both encruption and signing : to investigate